examples/fluids/problems/eulervortex.c

```diff
@@ -49,7 +49,7 @@ PetscErrorCode NS_EULER_VORTEX(ProblemData *problem, DM dm, void *ctx, SimpleBC
   problem->apply_outflow.qfunction_loc       = Euler_Outflow_loc;
   problem->bc                                = Exact_Euler;
   problem->bc_ctx                            = euler_ctx;
-  problem->non_zero_time                     = PETSC_TRUE;
+  problem->non_zero_time                     = PETSC_FALSE;  // TODO: add support for converting state variables
```

Comment:
Could this be split into two flags, say `non_zero_time_cons` and `non_zero_time_prim`, so we can still report errors in the state variables, skipping an attempt to convert?

Reply:
Having a second flag makes sense but how about using `problem->convert_state.qfunction` for that purpose?